mypy/semanal.py

```diff
@@ -957,16 +957,16 @@ def analyze_property_with_multi_part_definition(self, defn: OverloadedFuncDef) -
         deleted_items = []
         for i, item in enumerate(items[1:]):
             if isinstance(item, Decorator):
-                if len(item.decorators) == 1:
+                if len(item.decorators) >= 1:
                     node = item.decorators[0]
```

Comment:
the first item must be the setter - any other decorator is after the setter:

good:
```
@foo.setter
@dec
def foo(self, ...):
    ...
```
bad:
```
@dec
@foo.setter
def foo(self, ...):
    ...
```

Reply:
"@overload" support is still broken:

```
from typing import TypeVar, Generic, overload
from typing_extensions import Self

T_str_or_bytes = TypeVar("T_str_or_bytes", bound=str | bytes)


class Foo(Generic[T_str_or_bytes]):
    def __init__(self, v: T_str_or_bytes) -> None:
        self._v = v

    @property
    @overload
    def ham(self: Foo[str]) -> str:
        return self._v

    @property
    @overload  # E: Decorated property not supported
    def ham(self: Foo[bytes]) -> bytes:
        return self._v
```